src/components/CustomAnimatedModal.tsx

```diff
@@ -7,54 +7,50 @@ import { RequestSwapStack } from '../navigation/RequestSwap/RequestSwapStack';
 interface CustomAnimatedModalProps {
     isVisible: boolean;
```

Comment:
this is no longer being used :( I left it cause there might be a need for it in the future?

Reply:
makes sense to me. modals will need to be hidden/shown as needed. 
oh wait, you're talking about the whole file. I see that both `CustomAnimatedModal` or `CustomAnimatedModalProps` are currently only imported but not used in your `swap-v2` branch. 
i think let's keep it for now, unless you happen have time to find a built-in or plugin for modals.